partcad-cli/src/partcad_cli/click/conftest.py

```
@@ -0,0 +1,21 @@
+import pytest
+from click.testing import CliRunner, Result
+from typing import Iterator
```

Ruff (0.8.2)

Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)